mypy/types.py

```diff
@@ -2405,6 +2405,19 @@ def remove_optional(typ: Type) -> ProperType:
         return typ
 
 
+def is_literal_type(typ: Type, fallback_fullname: str, value: LiteralValue) -> bool:
+    """Returns 'true' if this type is a LiteralType with the given value
+    and underlying base fallback type.
+    """
```

Comment:
It would be easy to fit this on a single line if you phrase this as `Check if this type is a LiteralType with the given value and fallback type.`